weblate/templates/mail/account_activity.html

```diff
@@ -31,6 +31,7 @@
 <p>
 {% blocktrans %}If you don’t recognize this change, you can report it by clicking the button below.{% endblocktrans %}
 <b>{% blocktrans %}Don’t use this button for support questions. Such questions won’t be answered.{% endblocktrans %}</b>
+{% blocktrans %}If you have an idea or a question, <a href="https://github.com/WeblateOrg/weblate/issues/new/choose">this</a> is a place to start.{% endblocktrans %}
```

Comment:
Better to keep the link outside the translated text - it can change over the time:

```suggestion
{% blocktrans with link_start='<a href="https://github.com/WeblateOrg/weblate/issues/new/choose">' link_end='</a>'  %}If you have an idea or a question, {{ link_start }}this{{ link_end }} is a place to start.{% endblocktrans %}
```